qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -111,13 +143,15 @@ def sample(self, problem: BaseProblem, points: List[float]) -> BOPESSamplerResul
         """
         self._problem = problem
         self._driver = problem.driver
+        self._aux_operators = aux_operators
```

Comment:
Is it really necessary to store these on `self`? It should suffice to pass them into `evaluate_callable_aux_operator`

Reply:
The method sample() calls run_points() which calls run_single_points() which calls solver.solve() on the problem and the aux_operators. Instead of passing the arguments aux_operators through this chain of methods, I suggested that aux_operators would be stored in self.